package.json

```diff
@@ -42,5 +42,17 @@
         "path": "./themes/OneNord-color-theme.json"
       }
     ]
+  },
+  "devDependencies": {
+    "js-yaml": "^4.1.0",
+    "prettier": "^3.0.0",
+    "vsce": "^1.97.0"
+  },
+  "scripts": {
+    "build": "node ./scripts/build.js",
+    "lint": "node ./scripts/lint.js",
+    "package": "vsce package -o ./bin/one-nord.vsix",
+    "vscode:prepublish": "npm run build",
```

Comment:
Probably you want to link as well before publishing and if it fails to stop building and publishing